src/OrchardCore.Modules/OrchardCore.Users/Permissions.cs

```diff
@@ -8,16 +8,18 @@ public class Permissions : IPermissionProvider
     {
         public static readonly Permission ManageUsers = CommonPermissions.ManageUsers;
         public static readonly Permission ViewUsers = CommonPermissions.ViewUsers;
-
-        public static readonly Permission ManageOwnUserInformation = new Permission("ManageOwnUserInformation", "Manage own user information", new Permission[] { ManageUsers });
+        public static readonly Permission ManageOwnUserInformation = new("ManageOwnUserInformation", "Edit own user information", new Permission[] { ManageUsers });
```

Comment:
yes I do not like the name either. but do not want to change it to avoid a breaking change. `ManageOwnUserInformation` is now implied by `EditUser`

Reply:
Need to be also implied by `DeleteUsers`.

Reply:
Why do we need to do that?

Reply:
About implied by DeleteUsers, forgot

Then I suggested to also define ManageOwnUserInformation in CommonPermissions as ManageUsers, ViewUsers, EditUsers and so on are e.g. `CommonPermissions.EditUsers`.

But as you want as it seems that `ManageOwnUserInformation` was already only defined here.

Reply:
I added `EditOwnUser` permission to the `CommonPermissions` role and pointed `PermissionsManageOwnUserInformation` to it.  the static properties in `Permissions` class are not used by the project but will leave them to prevent breaking change. Maybe we should add `[Obsolete()]` attribute on them so we can remove them in future releases.